src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3RequestStream.cs

```diff
@@ -1196,7 +1199,7 @@ private void HandleReadResponseContentException(Exception ex, CancellationToken
                     // Our connection was reset. Start aborting the connection.
                     Exception abortException = _connection.Abort(ex);
                     throw new IOException(SR.net_http_client_execution_error, new HttpRequestException(SR.net_http_client_execution_error, abortException));
-                case Http3ConnectionException:
+                case HttpProtocolException:
```

Comment:
Judging by `CreateHttp3StreamException` in `HttpProtocolException` it seems to me like `HttpProtocolException` may not always be connection-level. Is that right? This line does not seem as equivalent change then. Or am I missing something?

Reply:
The HttpProtocolException thrown from stream is not on a code path which would affect this. In other words, only connection-level exceptions can be encountered here